sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -30,13 +31,110 @@ export interface BeginSearchAvailablePhoneNumbersOptions extends OperationOption
 export interface BeginUpdatePhoneNumberCapabilitiesOptions extends OperationOptions {
 }
 
+// @public (undocumented)
+export interface Consent {
```

Comment:
For a response model this has a lot of optional properties, should they be required?

Reply:
Good point, i dont think so, checking with service team,